clang/test/dpct/cmake_migation/case_015/expected.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.24)
+project(AProject)
+cuda_add_library(target_one MODULE nosrc1.cu)
+
+# Test C++ 99 standard
+set_target_properties(target_one PROPERTIES CXX_STANDARD 17)
+
+# Test C++ 99 standard
+set_target_properties(target_one PROPERTIES CXX_STANDARD 17)
+
+# Test C++ 99 standard
+set_target_properties(target_one PROPERTIES CXX_STANDARD 17)
+
+# No change unless property matches exactly
+set_target_properties(target_one
+                      PROPERTIES
+                      CXX_Standard 14)
```

Comment:
Is CXX_Standard a invalid cmake syntax?

Reply:
`CXX_Standard` is not a CMake defined variable.